Source/Scene/ModelExperimental/ModelExperimentalPrimitive.js

```diff
@@ -126,7 +127,8 @@ function initialize(runtimePrimitive) {
 
   var hasFeatureIds =
     defined(primitive.featureIdAttributes[featureIdAttributeIndex]) ||
-    defined(primitive.featureIdTextures[featureIdTextureIndex]);
+    defined(primitive.featureIdTextures[featureIdTextureIndex]) ||
+    defined(content.featureMetadata);
```

Comment:
This check fails in unit tests if `content` is undefined. check that content exists first.